workflows/export/shorts.go

```diff
@@ -0,0 +1,500 @@
+	}
+
+	return nil
+}
+
+// ExportShort exports a single short to BCCM Platform
+func ExportShort(ctx workflow.Context, short *ShortsData) error {
+	logger := workflow.GetLogger(ctx)
+	logger.Debug("Starting export for %s", short.MBMetadata.ID)
+
+	tempFolder, err := wfutils.GetWorkflowTempFolder(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get temp folder: %w", err)
+	}
+
+	err = triggerShortExport(ctx, short)
+	if err != nil {
+		return fmt.Errorf("failed to trigger export: %w", err)
+	}
+
+	res, err := wfutils.Execute(ctx, activities.Vidispine.GetFileFromVXActivity, vsactivity.GetFileFromVXParams{
+		VXID: short.MBMetadata.ID,
+		Tags: []string{"original"},
+	}).Result(ctx)
+	if err != nil {
+		return err
+	}
+
+	short.OriginalPath = &res.FilePath
+
+	thumb, err := generateThumbnailForShort(ctx, tempFolder, short)
+	if err != nil {
+		return fmt.Errorf("failed to generate thumbnail: %w", err)
+	}
+
+	_, styledImage, err := uploadImage(ctx, true, "poster", thumb)
+	if err != nil {
+		return fmt.Errorf("failed to upload thumbnail: %w", err)
+	}
+
+	err = createShortInPlatform(ctx, short, styledImage)
+	if err != nil {
+		return err
+	}
+
+	// Update Notion status to "Done" via activity
+	if short.CSV.NotionPageID != "" {
+		err = wfutils.Execute(ctx, activities.Notion.UpdateAssetStatus, activities.UpdateAssetStatusArgs{
+			RowID:  short.CSV.NotionPageID,
+			Status: "Done",
+		}).Get(ctx, nil)
+		if err != nil {
+			logger.Error("Failed to update Notion status via activity", "error", err, "rowId", short.CSV.NotionPageID)
+		}
+	} else {
+		logger.Error("No NotionPageID present for short, skipping Notion status update", "mb_id", short.MBMetadata.ID)
+	}
+
+	return nil
+}
+
+func createShortInPlatform(ctx workflow.Context, short *ShortsData, styledImage *directus.StyledImage) error {
+
+	// Create mediaitem
+	language := short.CSV.Language
+	if language == "" {
+		language = "no"
+	}
+
+	parentStartsAt, parentEndsAt, err := getInOutTime(short)
+	if err != nil {
+		return err
+	}
+
+	// Get asset for item
+	assetResult, err := wfutils.Execute(ctx, activities.Directus.GetAssetByMediabankenID, short.MBMetadata.ID).Result(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get asset: %w", err)
+	}
+
+	if assetResult == nil {
+		return fmt.Errorf("no asset found for mediabanken_id: %s", short.MBMetadata.ID)
+	}
+
+	assetID := strconv.Itoa(int(assetResult.ID))
+
+	var episodeID string
+	if short.CSV.EpisodeID == "" {
+		episodeID = short.CSV.EpisodeID
+	} else {
+		fmt.Printf("WARN: EpisodeID is empty for %s, %s\n", short.MBMetadata.ID, short.CSV.Label)
+	}
+
+	label := short.CSV.Label
+	if label == "" {
+		label = short.MBMetadata.Get(vscommon.FieldTitle, "<NO TITLE>")
+	}
+
+	// Create media item
+	mediaItemResult, err := wfutils.Execute(ctx, activities.Directus.CreateMediaItem, activities.CreateMediaItemInput{
+		Label:           label,
+		Type:            "short",
+		AssetID:         assetID,
+		Title:           "",
```

Comment:
Prob should have a title

Reply:
Considering the original is this way (https://github.com/BCC-Media/utils/blob/c8a98f345a5d04096d4ccbe829d747a7b474a73a/go/scripts/import-shorts.go#L230) I'll leave it as is and change on demand.